### ISSUE TITLE
Always print “The qiskit parameterization bug is already fixed!” when running mnist_example.py

### ISSUE THREAD

**Issue by ex193170** [mit-han-lab/torchquantum#14]
I tried to run the `minist_example.py`. And I commented out the later part about Qiskit.

I tried to run it with the following command:

`python mnist_example.py --epochs 1`

But always print "The qiskit parameterization bug is already fixed!" in the terminal. I wish I had a way to stop printing, but I haven't found one yet.

Thank!

[print_log.txt](https://github.com/mit-han-lab/torchquantum/files/8456994/print_log.txt)

**Comment by Hanrui-Wang** [comment]
Hi Ke Chen,

Thank you for your feedback! We have fixed it, please pull the latest version of torchquantum. Alternatively, you can comment out the line 32 in file `torchquantum/__init__.py`

`print('The qiskit parameterization bug is already fixed!')`

Thanks!
Hanrui

**Comment by ex193170** [comment]
Think! I tried again to run the `minist_example.py` in the `examples/simple_mnist`. This issue has been fixed.

But I still have a small issuse. I read this article ([HPCA'22] QuantumNAS: Noise-Adaptive Search for Robust Quantum Circuits), I have benefited a lot. I think it's a great job.

However, I don't understand some details, so I want to see the code to help me understand. I find it in `artifact/example2`. The shell scripts inside this folder use `examples/train.py`, `examples/run_es.py` and `examples/eval.py`, but I didn't find them. So I'm wondering if it's because the code isn't open source?
